test/testdata/workspaces/python-flask-sample-app/sonar-project.properties

```diff
@@ -0,0 +1,18 @@
+# Comma-separated paths to directories with sources (required)
+sonar.sources=src
+
+# Comma-separated paths to directories containing test source files.
+sonar.tests=tests
+sonar.test.inclusions=**/*_test.py
+
+# Encoding of the source files (optional but recommended as default is ASCII)
+sonar.sourceEncoding=UTF-8
+
+# Exclude test files from coverage analysis
+sonar.coverage.exclusions=tests/**
+
+# Define unit test information inside the python project
+sonar.python.xunit.reportPath=build/test-results/test/report.xml
```

Comment:
if the path is changed as per previous comment then we should change it here too